vtr_flow/scripts/python_libs/vtr/task.py

```diff
@@ -473,56 +476,45 @@ def create_jobs(args, configs, after_run=False) -> List[Job]:
     """
     jobs = []
     for config in configs:
-        for arch, circuit in itertools.product(config.archs, config.circuits):
-            noc_traffic = []
-            if config.noc_traffics:
-                noc_traffics = config.noc_traffics
-            else:
-                noc_traffics = [None]
-            for noc_traffic in noc_traffics:
-                golden_results = load_parse_results(
-                    str(PurePath(config.config_dir).joinpath("golden_results.txt"))
+        if config.noc_traffic_list_type == "outer_product":
```

Comment:
I'd comment what you are doing here. Many options in task.py expand to a cartesian product, but we don't always want that for NoC traffic flows.